cmake/CMakeLists.txt

```diff
@@ -636,3 +636,6 @@ if (onnxruntime_BUILD_CSHARP)
   include(onnxruntime_csharp.cmake)
 endif()
 
+if (NOT (${CMAKE_SYSTEM_NAME} MATCHES "Android"))
+  include (onnxruntime_external_ops.cmake)
```

Comment:
this isn't optional?
we are really embedding python into core onnxruntime by default?

Reply:
Yeah, we should make this optional using build flags.

Reply:
Build flag added.